com.deezer.api.track/request

```diff
@@ -0,0 +1,2 @@
+GET https://api.deezer.com/search/track/?output=json&strict=on&q={{q}}
+  Accept "application/json"
```

Comment:
For the Accept header, you should remove the spaces at the beginning of the line and use this format:

```
Accept: application/json
```